src/psi-behavior.scm

```diff
@@ -118,22 +118,21 @@
 	(DefinedPredicate "Say whoa!")
 	speech-demand-satisfied (stv 1 1) speech-demand)
 
-; XXX FIXME
-; The following two rules cause the (DefinedPredicate "Normal:amused")
-; to run as fast as possible -- something like 30 times a second.
-; Something is wrong here, see bug
-; opencog/ros-behavior-scripting/issues/110
-;(psi-rule (list (DefinedPredicate "very low sound?"))
-;	(DefinedPredicate "Quiet:happy")
-;	speech-demand-satisfied (stv 1 1) speech-demand)
-;
-;(psi-rule (list (DefinedPredicate  "normal conversation?"))
-;	(DefinedPredicate "Normal:amused")
-;	speech-demand-satisfied (stv 1 1) speech-demand)
-
+(psi-rule (list (DefinedPredicate  "saliency required?"))
+   (DefinedPredicate "Salient:Curious")
+   face-demand-satisfied (stv 1 1) face-demand)
+   
+(psi-rule (list (DefinedPredicate "Room bright?"))
+   (DefinedPredicate "Bright:happy")
+   face-demand-satisfied (stv 1 1) face-demand)
+
+(psi-rule (list (DefinedPredicate "Heard Something?"))
+   (DefinedPredicate "React to Sound")
+   speech-demand-satisfied (stv 1 1) speech-demand)
```

Comment:
another example of mixed tab-space indentation. This ends up looking crazy for anyone who has different tab stops